client/src/entrypoints/admin/telepath/widgets.js

```diff
@@ -180,15 +180,15 @@ class DraftailInsertBlockCommand {
     this.split = split;
 
     this.blockMax = addSibling.getBlockMax(blockDef.name);
-    this.icon = `#icon-${blockDef.meta.icon}`;
+    this.icon = blockDef.meta.icon;
     this.description = blockDef.meta.label;
     this.type = blockDef.name;
   }
 
   render({ option }) {
     // If the specific block has a limit, render the current number/max alongside the description
     const limitText =
-      typeof blockMax === 'number'
+      typeof this.blockMax === 'number'
```

Comment:
Unrelated bug fix – I’m surprised this isn’t being picked up by linting. Without `this.`, this code can’t work.